client/src/pages/inventory/ActivityCreate.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useMutation } from 'react-query';
+import { createSearchParams, useNavigate } from 'react-router-dom';
+
+import { Alert, AlertTitle, Container } from '@mui/material';
+
+import { createActivity } from '../../data';
+import { PostActivityBody } from '../../types';
+
+import PageHeader from '../../components/page-header/PageHeader';
+import Spacer from '../../components/spacer/Spacer';
+import ActivityBase from './ActivityBase';
+
+const ActivityCreate = () => {
+  const navigate = useNavigate();
+
+  const { isLoading, isError, mutate } = useMutation(createActivity, {
+    onSuccess: () => {
+      navigate({
+        pathname: '/inventory',
+        search: createSearchParams({
+          tab: '1',
```

Comment:
I would use more explicit search name like `product` or `inventory` instead of 0-1